src/components/card/CardHole.jsx

```diff
@@ -1,7 +1,27 @@
+// @flow strict
 import React from 'react';
-import PropTypes from 'prop-types';
+import type {Node} from 'react';
 import classnames from 'classnames';
 
+export type ColorType =
+  | 'blue'
```

Comment:
maybe we should have generic ColorType with all colors? WDYT?

Reply:
maybe at least a few such types - different components allows only a subset of colors we have, not all of them